resources/wikia/modules/tracker.js

```diff
@@ -46,7 +46,7 @@
 			],
 			// @see /extensions/wikia/AnalyticsEngine/js/universal_analytics.js
 			guaTrackEvent = window.guaTrackEvent,
-			guaTrackAdEvent = window.guaTrackAdEvent,
+			guaTrackAdEvent = window.guaTrackAdEvent || (window.Mercury && window.Mercury.Modules.Ads.gaTrackAdEvent),
```

Comment:
There is:
```
// Required by ads tracking code
window.gaTrackAdEvent = Ads.gaTrackAdEvent;
```
in https://github.com/Wikia/mercury/blob/dev/front/common/modules/Ads.js#L72.

Shouldn't we use `window.gaTrackAdEvent` here?